renovate.json

```
@@ -20,6 +20,14 @@
       "fileMatch": ["^.github/workflows/continuous-integration.yml$"],
       "matchStrings": ["(?<depName>cypress/included):(?<currentValue>[0-9.]+)\n"],
       "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": ["^renovate.json$"],
+      "matchStrings": [
+        "\"node\": \"= (?<latestVersion>.+?)"
```

```suggestion
        "\"node\": \"= (?<currentValue>[^\"]+)\""
```
Ich glaube du hast das schliessende Anführungszeichen vergessen, und ausserdem muss die capture group `currentValue` heissen.